content/guides/identity-management/quick-start/5-find-or-create-box-users.md

```diff
@@ -0,0 +1,499 @@
+---
+type: quick-start
+hide_in_page_nav: true
+---
+
+# Find or create Box app users
+
+At this point we have application code that will handle traffic from users
+visiting, forward them to Okta to login, provide Okta user information, before
+finally handing off to a yet to be created handler for Box.
+
+This section will cover the final Box components: 
+
+* Validating if an Okta user has an associated Box app user account.
+* Creating a new app user for the associated Okta record if they don't.
+* Fetching tokens for the Box user to make user-specific API calls.
+
+## Create New App Users
+
+Before validating users we need a method for creating an associated Box user
+account if one doesn't already exist for the Okta user.
+
+<Choice option='programming.platform' value='node' color='none'>
+
+In your local application directory, load the `server.js` file created in
+step 1.
+
+Add the following `box` object into the file and save. 
+
+```js
+  const box = (() => {
+    const configJSON = JSON.parse(fs.readFileSync(path.resolve(__dirname, './../config.json')));
+    const sdk = boxSDK.getPreconfiguredInstance(configJSON);
+    const client = sdk.getAppAuthClient('enterprise');
+
+    let oktaRecord = {};
+    let userId = '';
+    let userClient;
```

Comment:
These 3 variables are not actually used according to my code editor.

Reply:
They're used once the individual functions (validateUser / createUser) have their code in them. They used shared instances of those variables.